dashboard/app/controllers/reference_guides_controller.rb

```diff
@@ -68,6 +77,13 @@ def edit
 
   private
 
+  def find_last_child(course_version_id, parent_key)
```

Comment:
It seems like you are always finding the last child to find the position value and add 1 to it. Would it be better to just make this method return the next open position?

Reply:
will do!